<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LimitEvaluatorCreatorWidget</class>
 <widget class="QWidget" name="LimitEvaluatorCreatorWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>341</width>
    <height>108</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="DataTypeLabel">
     <property name="text">
      <string>Data type:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="DataTypeComboBox"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="LimitValueLabel">
     <property name="text">
      <string>Limit value:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="LimitValueLineEdit"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="EvalOpLabel">
     <property name="text">
      <string>Evaluation operation:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QComboBox" name="EvalOpComboBox"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
